config/components/cilium-operator.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "cilium-operator"
```

Comment:
If vendor and application would be `cilium` for both components according to the sample CVE you provided, then this value should be `cilium` as well

Reply:
done at https://github.com/bitnami/vulndb/pull/441/commits/e98aa5313ff162656a7c6266ae27453bde2c4e8c